src/DegenGambit.sol

```diff
@@ -580,14 +580,17 @@ contract DegenGambit is ERC20, ReentrancyGuard {
                 // 3 of a kind with a major symbol. Jackpot!
                 result = address(this).balance >> 1;
             }
-        } else if (left == right) {
+        } else if (left == right && center >= 16) {
             // Outer pair combos.
-            if (left >= 1 && left <= 15 && center >= 16) {
+            if (left >= 1 && left <= 15) {
                 // Minor symbol pair on outside reels with major symbol in the center.
                 result = 100 * CostToSpin;
                 if (result > address(this).balance >> 4) {
                     result = address(this).balance >> 4;
                 }
+            } else if (left >= 16) {
```

Comment:
Doesn't the next case handle this?

```solidity
        } else if (
            left >= 16 &&
            right >= 16 &&
            center >= 16 &&
            left != center &&
            right != center
        ) {
```

Reply:
No - because of `else`. Nice work @mrk-hub !

I suggest removing all nesting and cleverness from the conditions. Let's make each branch match exactly one condition. What do you think?

Reply:
I think that works well and it'll make it easier for code viewers to understand what payout scenario if being processed.